### PR TITLE
Master list of project-wide issue labels

### PR DESCRIPTION
**Pull request by texasmichelle**
From #12, for use in #24.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/37)

### CODE REVIEW

File: issue-labels.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+|             | other
+| priorities  | p0
+|             | p1
+|             | p2
+| problems    | bug
+|             | refactor
+|             | security
+| status      | icebox
+|             | backlog
+|             | in progress
```
Comment (ankushagarwal) [inline issue-labels.md]: Why use whitespace in `in progress` when hypens are used for other labels

Comment (texasmichelle) [reply issue-labels.md]: Good catch

**Pull request merged**